tests/layer_tests/requirements.txt

```diff
@@ -4,6 +4,7 @@ numpy
 requests
 torch
 torchvision
+transformers
```

Comment:
do you really need install transformers in this PR?
I believe it is related to another PR

Reply:
I do, I added test for primitive from transformers